docker-compose/readme.md

```diff
@@ -462,37 +462,20 @@ You can run selenium tests against a local bb2 server by following below steps:
    ./docker-compose/run_selenium_tests_local.sh account
    ```
 
-3. To debug tests (visualize browser interaction): point VNC client to localhost:5900
```

Comment:
Can we no longer do this?

Reply:
We took out the `-d` option in #1180, so yea if we need to reintroduce this ability, we'd need to figure out a way to do so that is usable on Mac.

Reply:
That's totally fine. We can circle back to that as a separate piece of work if it feels like we need to.